src/auth/none/AuthNoneProtocol.h

```diff
@@ -22,7 +22,7 @@ class CephContext;
 struct AuthNoneAuthorizer : public AuthAuthorizer {
   AuthNoneAuthorizer() : AuthAuthorizer(CEPH_AUTH_NONE) { }
   bool build_authorizer(const EntityName &ename, uint64_t global_id) {
-    __u8 struct_v = 1;
+    __u8 struct_v = 1; // see AUTH_MODE_* in Auth.h
```

Comment:
The second AUTH_MODE_ value I put at 100 to leave room.  I'll fix up the detection checks to look for a range of values.

Reply:
Yeah I'm more concerned that I think the detection checks were requiring it to be 1 — or perhaps just the 1-bit? Either way, it will break if/when we switch to struct_v=2.